examples/spear_qcp/SMAC4AC_spear_qcp.py

```diff
@@ -23,6 +23,7 @@
                           # non-deterministic target algorithm
                           'initial_budget': 1,
                           'eta': 3,
+                          'min_chall': 1  # because successive halving cannot handle min_chall > 1
                           }
 
     smac = SMAC4AC(scenario=scenario,  # scenario object
```

Comment:
Wouldn't it be good if you used the BOHB facade here?

Reply:
I wanted to show some example that can use SMAC4AC facade and use that to pass a different intensifier. So I combined them. Also, isn't the BOHB facade only for HPO?

Reply:
> python -c 'import autosklearn.data.competition_c_functions; print("SO imported from: %s" % autosklearn.data.competition_c_functions.__file__)'

That's a good question, maybe we should leave SMAC4AC then.